api/game.go

```diff
@@ -48,7 +48,10 @@ func (g *Game) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // GET /api/game/map.json
 func (g *Game) ServeMap(w http.ResponseWriter, r *http.Request) {
+	g.players.mutex.Lock()
 	JSON, err := json.Marshal(g)
```

Comment:
json.Marshal will only JSON-ify public fields of game; so these are Min, Max, Width, and Height. There aren't mutexes for those fields as they are read-only, so this is a redundant lock

Reply:
(Apart from one HTTP route that changes them, but that's expected to be run before the game starts, so no mutex is really needed)

Reply:
> will only JSON-ify public fields of game

oh, wild!

Reply:
Go is really interesting in this way - capitalization differentiates public and private fields 🫠

Reply:
I'm not sure how that makes me feel haha